apps/site/lib/site_web/views/date_time_selector.ex

```diff
@@ -12,6 +12,26 @@ defmodule DateTimeSelector do
                     int -> Integer.to_string(int)
                   end)
 
+  @spec verify_params(map, map) :: map
+  defp verify_params(form, datetime) do
+    default_datetime_params = %{
+      "year" => datetime.year,
+      "month" => datetime.month,
+      "day" => datetime.day,
+      "hour" => datetime.hour,
```

Comment:
I'm seeing some funny defaults, like 13 (hour) AM.

Reply:
This default datetime corresponds with `Util.now()` on first load but after I click the button to plan a trip it becomes
```
%{
  "am_pm" => "AM",
  "day" => 21,
  "hour" => 13,
  "minute" => 20,
  "month" => 12,
  "year" => 2020
}
```
(when I run it now at 6:20PM in this time zone)

This probably isn't caused by your PR but something sinister is going on here...

Reply:
I think the latest changes will not cause any inconsistencies (I was not converting the time properly).
Were you entering specific values in the Trip Planner so I can reproduce the issue you were having?